### PR TITLE
Added monitors information to 1Password app doc

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to add monitors and its respective alerts details to 1Password app doc.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[Update the doc for the existing 1Password App](https://sumologic.atlassian.net/browse/DOCS-398)

### CODE REVIEW

File: docs/integrations/1password.md [hunk 1]
```diff
@@ -153,3 +153,21 @@ The **1Password - Threat Intel** dashboard provides high-level views of threats
...
+
+import CreateMonitors from '../reuse/apps/create-monitors.md';
+
+<CreateMonitors/>
+
+### 1Password alerts
+
+| Name | Description | Trigger Type (Critical / Warning / MissingData) | Alert Condition | 
+|:--|:--|:--|:--|
+| `Privileged vault and group access control changes` | This alert is fired when there is changes to privileged vault and group access control. | Critical | Count > 0 |
```
Comment (jpipkin1) [inline docs/integrations/1password.md:167]:
```suggestion
| `Privileged vault and group access control changes` | This alert is fired when there are changes to privileged vault and group access control. | Critical | Count > 0 | 
```

File: docs/integrations/1password.md [hunk 2]
```diff
@@ -153,3 +153,21 @@ The **1Password - Threat Intel** dashboard provides high-level views of threats
...
+<CreateMonitors/>
+
+### 1Password alerts
+
+| Name | Description | Trigger Type (Critical / Warning / MissingData) | Alert Condition | 
+|:--|:--|:--|:--|
+| `Privileged vault and group access control changes` | This alert is fired when there is changes to privileged vault and group access control. | Critical | Count > 0 | 
+| `Login Exceeds Set Parameters` | This alert is fired when the user tries login activities from more than one location. | Critical | Count > 1|
+| `Grant User Vault Access` | This alert is fired when the user is granted access to a vault. | Critical | Count > 0 |
+| `Changes to a User's MFA settings` | This alert is fired when user makes changes in the MFA settings. | Critical | Count > 5 |
```
Comment (jpipkin1) [inline docs/integrations/1password.md:170]:
```suggestion
| `Changes to a User's MFA settings` | This alert is fired when a user makes changes in the MFA settings. | Critical | Count > 5 |
```

File: docs/integrations/1password.md [hunk 3]
```diff
@@ -153,3 +153,21 @@ The **1Password - Threat Intel** dashboard provides high-level views of threats
...
+| `Audit Events from Risky Locations` | This alert is fired when audit event is registered from Embargo location (Afghanistan, China, Cuba, North Korea, Iran, Libya, Nigeria, Sudan, Syria, and Yemen). | Critical | Count > 0 |
```
Comment (jpipkin1) [inline docs/integrations/1password.md:171]:
```suggestion
| `Audit Events from Risky Locations` | This alert is fired when an audit event is registered from an embargo location (Afghanistan, China, Cuba, North Korea, Iran, Libya, Nigeria, Sudan, Syria, and Yemen). | Critical | Count > 0 |
```

**Review by jpipkin1 — APPROVED** [review]
I have a few suggestions.

**Comment by JV0812** [comment]
Thanks for your feedback @jpipkin1. I have fixed your comments.

**Review by jpipkin1 — APPROVED** [review]

**Comment by jpipkin1** [comment]
I am merging the PR at the request of @ankitgoelcmu.

**Pull request merged**